packages/fast-usdc/test/cli/cli.test.ts

```diff
@@ -81,43 +81,23 @@ test('shows help for config show command', async t => {
   t.snapshot(output);
 });
 
-test('shows help for deposit command', async t => {
-  const output = await runCli(['deposit', '-h']);
-  t.snapshot(output);
-});
-
-test('shows help for withdraw command', async t => {
-  const output = await runCli(['withdraw', '-h']);
-  t.snapshot(output);
-});
-
 test('shows error when deposit command is run without options', async t => {
-  const output = await runCli(['deposit']);
+  const output = await runCli(['lp', 'deposit']);
```

Comment:
I wonder about demoting `deposit` and `withdraw` under `lp`. Didn't @turadg considered that option and decide against it?

Reply:
Reverted